feature/system/control_plane_traffic/otg_tests/default_copp_test/README.md

```diff
@@ -0,0 +1,178 @@
+We will not configure any CoPP policies or thresholds on the DUT. We will solely rely on the vendor default and pre-applied CoPP policies and thresholds \
+Vendor implementation may differ and all vendors may not support all the subtests in this readme \
+Use OC wherever applicable and available e.g. configuring interfaces
+[TODO] Utilize standardized OC models as they become available
+
+*   Ensure that the default CoPP protects the control plane from excessive traffic and prioritizes critical control plane packets
+*   Verify that the default CoPP queues function correctly and handle packet drops as expected
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Topology:
+
+```mermaid
+graph LR;
+A[ATE] <-- Port1(IPv4/6) --> B[DUT];
+B[DUT] <-- Port2(IPv4/6) --> C[ATE];
+```
+
+## Procedure
+
+### Initial setup
+
+*   Connect DUT port-1, 2 to ATE port-1, 2 respectively
+*   Configure IPv4 and IPv6 addresses on DUT and ATE ports as shown below (using OC)
+
+    *   DUT port-1 IPv4 address ```dp1-v4 = 192.168.1.1/30```
+    *   ATE port-1 IPv4 address ```ap1-v4 = 192.168.1.2/30```
+
+    *   DUT port-2 IPv4 address ```dp2-v4 = 192.168.1.5/30```
+    *   ATE port-2 IPv4 address ```ap2-v4 = 192.168.1.6/30```
+
+    *   DUT port-1 IPv6 address ```dp1-v6 = 2001:DB8::1/126```
+    *   ATE port-1 IPv6 address ```ap1-v6 = 2001:DB8::2/126```
+
+    *   DUT port-2 IPv6 address ```dp2-v6 = 2001:DB8::5/126```
+    *   ATE port-2 IPv6 address ```ap2-v6 = 2001:DB8::6/126```
+
+
+### SYS-1.1.1 - Test CoPP for BGP Traffic
+
+*   Configure BGP between ATE Port-1 and DUT port-1
+*   Generate BGP traffic (update/keepalive) from ATE port-1 within the system defined thresholds
+*   Generate BGP traffic (update/keepalive) from ATE port-1 exceeding the system defined thresholds
+* Monitor CoPP telemetry for the BGP traffic class using the best available method to verify that:
+    *   ***[TODO]*** **gNMI/OC:** Query standardized OC CoPP state model and use OC path via gNMI
+        *   `/components/component/integrated-circuit/pipeline-counters/control-plane-traffic/vendor/..`
+    *   **CLI fallback**: Execute vendor-specific CLI (like show cpu counters queue summary or more specific CoPP counter commands) and parse the output.
+    *   Packets exceeding the threshold are dropped for each queue
+    *   Packets within the threshold are not dropped
+    *   BGP Session remains stable throughout the test
+    *   Analyze the CPU utilization to ensure it remains stable under traffic stress
+
+### SYS-1.1.2 - Test L3 destination missing and Longest Prefix Match (LPM) overflow counters
+
+*   Generate traffic from ATE port-1 for destination IP addresses using `172.16.0.0/16` range
+*   The route for these destinations will/should not be present on DUT
+*   Generate traffic within the system defined thresholds and exceeding them
```

Comment:
since we don't know the system thresholds (they are some factory default), instead send a very high amount of traffic (near wire speed) with no ip route.   Then verify bgp sessions stay and and CPU utilization can be fetched and logged.

Reply:
This line needs to be changed to something like, send 100,000 pps on tcp/179 to the device.   

As it reads now, somehow we have to figure ou what the system defined thresholds are, and we don't know this / don't want to add vendor specific constraints into this test.